napari/utils/colormaps/colormap.py

```diff
@@ -350,17 +372,29 @@ def shuffle(self, seed: int):
         self.events.colors(value=self.colors)
 
 
+LabelColormap = deprecated_class_name(
+    CyclicLabelColormap,
+    'LabelColormap',
+    version='0.5.0',
+    since_version='0.4.19',
+)
+
+
 class DirectLabelColormap(LabelColormapBase):
     """Colormap using a direct mapping from labels to color using a dict.
 
     Attributes
     ----------
     color_dict: dict from int to (3,) or (4,) array
         The dictionary mapping labels to colors.
+
     use_selection : bool
-        Whether to color using the selected label.
+        Whether to map only the selected label to a color.
+        If `True` only selected label will be not transparent.
     selection : int
         The selected label.
+    colors : ColorArray
+        Exist because of implementation details. Please do not use it.
     """
```

Comment:
As background is used for validation of color_dict then it should be documented.